docs/data-sources/4via6.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
This documentation is lacking the example usage so it seems you may need to regenerate the documentation or double check you've created the example file correctly.

Reply:
👍🏻 `a173b1b`